src/app/game-state/activity.service.ts

```diff
@@ -619,27 +631,27 @@ export class ActivityService {
       level: 0,
       name: ['Attach Chains to the Island'],
       activityType: ActivityType.AttachChains,
-      description: ['Swim deep and attach one of your chains to the island.'],
-      consequenceDescription: ['Uses 1000 Stamina. Requires an unbreakable chain.'],
+      description: ['Swim deep and attach one of your chains to the island, then pull.'],
+      consequenceDescription: ['Uses 1000000 Stamina when carrying something REALLY heavy to try hauling something really REALLY heavy. You\'re going to need an Unbreakable Chain and a long time to rest.'],
       consequence: [() => {
-        this.characterService.characterState.status.stamina.value -= 1000;
-        if (this.inventoryService.consume("chain") > 0){
-          this.logService.addLogMessage("You attach a chain to the island. and give your chains a tug.","STANDARD","EVENT");
+        if (this.inventoryService.consume("chain") > 0 ){
+          this.characterService.characterState.status.stamina.value -= 1000000;
```

Comment:
Does this let them succeed even if they don't have the stamina?

Reply:
Crap, forgot to add that as a check since the old one didn't have that, definitely should. Had autorest on for testing like a derp.

Reply:
Hrm, now I'm thinking of making Pull Chains its own thing but meh, kinda feels extra.